syntax = "proto3";

message serviceMethod {
  string service_name = 1;
  string method_name = 2;
}
